service/src/main/resources/azure/marketplaceTemplate/mainTemplate.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Cool, thanks for checking this in. Just curious, did you successfully make a managed app definition from this (empty) template?

The template I used had a few more things in it, but might not actually be needed for WSM.

Reply:
Yes, I did make a managed app definition from that, but it probably doesn't make sense to have it in the WSM code base. I will remove it.